google_analytics/aggregation.py

```diff
@@ -1,7 +1,138 @@
+import json
 import os
+import shutil
+import glob
+import re
 import warnings
-import numpy as np
+
 import pandas as pd
+import numpy as np
+
+
+def reduce_df(path, output, nrows=None, chunksize=20000):
+    """ Load Google analytics data from JSON into a Pandas.DataFrame. """
+    if nrows and chunksize:
```

Comment:
Small remark: this message is now not printed when nrows = None (i.e the whole dataset).

Reply:
True but I was too bored to fix. Will do!